schematic/models/validate_attribute.py

```diff
@@ -823,16 +830,16 @@ def list_validation(
         # white spaces removed.
         errors = []
         warnings = []
+        replace_null = True
 
         csv_re = comma_separated_list_regex()
 
-        rule_parts = val_rule.lower().split(" ")
-        if len(rule_parts) > 1:
-            list_robustness = rule_parts[1]
-        else:
-            list_robustness = "strict"
+        list_robustness = get_list_robustness(val_rule=val_rule)
+
+        if list_robustness == "like":
+            replace_null = False
 
-        if list_robustness == "strict":
+        elif list_robustness == "strict":
             manifest_col = manifest_col.astype(str)
 
```

Comment:
Is list_roboustness every somethign besides "like" or "strict" and if so are the defaults on lines 831, 832 and 833 the values you want here?

Reply:
I believe `list_robustness` should only be `strict` or `like`, but the user could have specified other rule modifiers like `error` for example.

And yes?